### PR TITLE
fixing the sigma value in example input files and adding comments to the ecutwfc keyword (Useful Information: lower the barrier for users who is not familiar with 'ecut' in the INPUT file))

### LINKED ISSUE #5420: Docs: Some parameters' values set in our example might be misleading
### Details

There are users followed the way to set parameters and complained the result was unreliable. More annotations should be added to guide user to know what kind of parameter setting on one specific calculation task can lead to a reliable result.

### Have you read the online manual http://abacus.deepmodeling.com/en/latest/

- [x] Yes, I have read the online manual.

### Task list for Issue attackers (only for developers)

- [ ] Identify the specific section of the documentation with the issue.
- [ ] Investigate the issue and determine the root cause.
- [ ] Research best practices and potential solutions for the identified issue.
- [ ] Update the documentation to address the issue, following the suggested improvement.
- [ ] Ensure the updated documentation adheres to the project's documentation standards.
- [ ] Test the updated documentation to ensure it is clear and accurate.
- [ ] Review and incorporate any relevant feedback from users or developers.
- [ ] Publish the updated documentation and notify the issue reporter.

### PR DESCRIPTION
**Pull request by BariumOxide13716**
### Reminder
- [x] Have you linked an issue with this pull request?
- [ ] Have you added adequate unit tests and/or case tests for your pull request?
- [ ] Have you noticed possible changes of behavior below or in the linked issue?
- [ ] Have you explained the changes of codes in core modules of ESolver, HSolver, ElecState, Hamilt, Operator or Psi? (ignore if not applicable)

### Linked Issue
Fix #5420 

### Unit Tests and/or Case Tests for my changes
No unit test is needed for the documentation changes made in this pull request.

### What's changed?
Some misleading information are changed, including:
1. The sigma values are changed to no greater than 0.015 for gaussian smearing.
2. Some comments are added to ecutwfc in `INPUT` files and `KPT` file to ask users to check convergence of results.

### Any changes of core modules? (ignore if not applicable)
not applicable.

### CODE REVIEW

File: examples/bravais_lattice/fcc_SiO2/INPUT [hunk 1]
```diff
@@ -3,8 +3,13 @@ INPUT_PARAMETERS
...
 #Parameters (Accuracy)
-ecutwfc			50
+ecutwfc			 60 ###Energy cutoff needs to be tested to ensure your calculation is reliable.[1]
 scf_thr         	1e-8
 scf_nmax		20
 basis_type		pw
 latname			fcc
+
+
+### [1] Energy cutoff determines the quality of numerical quadratures in your calculations.
```
Comment (mohanchen) [inline examples/bravais_lattice/fcc_SiO2/INPUT:13]: we only suggest testing the ecut for plane wave basis set, for lcao basis set, we recommend not testting it but just use the cutoff in LCAO basis sets.

Comment (BariumOxide13716) [reply examples/bravais_lattice/fcc_SiO2/INPUT:13]: This is done and I made a new commit. Could you please help me double check it?

Comment (mohanchen) [reply examples/bravais_lattice/fcc_SiO2/INPUT:13]: OK

**Review by mohanchen — APPROVED** [review]

**Comment by mohanchen** [comment]
Excellent PR because the developer update the INPUT file, making user more easily to understand the input parameters 'ecut'.

**Pull request merged**